psqlextra/compiler.py

```diff
@@ -132,9 +196,12 @@ def _rewrite_insert(self, sql, params, return_id=False):
             self.qn(self.query.model._meta.pk.attname) if return_id else "*"
         )
 
-        return self._rewrite_insert_on_conflict(
-            sql, params, self.query.conflict_action.value, returning
-        )
+        if hasattr(self.query, "conflict_action"):
```

Comment:
**question:** Why was this extra check added?

Reply:
You're right. Has been a leftover from the first attempt to use the conflict handler for all "INSERT" statements and is removed now.